environments/.caas/ui-meta/slurm-infra-manila.yml

```diff
@@ -0,0 +1,105 @@
+name: "slurm-manila-preview"
+label: "Slurm: Manila Integration Preview"
+description: >- 
+  Batch cluster running the Slurm workload manager, the Open 
+  OnDemand web interface, and custom monitoring.
+
+  Has /project and /home from CephFS/Manila.
+logo: https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/Slurm_logo.svg/158px-Slurm_logo.svg.png
+
+parameters:
+  - name: cluster_floating_ip
+    label: External IP
+    description: The external IP to use for the login node.
+    kind: cloud.ip
+    immutable: true
+
+  - name: compute_count
+    label: Compute node count
+    description: The number of compute nodes in the cluster.
+    kind: integer
+    options:
+      min: 1
+    default: 3
+
+  - name: compute_flavor
+    label: Compute node size
+    description: The size to use for the compute node.
+    kind: "cloud.size"
+    immutable: true
+    options:
+      min_ram: 2048
+      min_disk: 20
+  
+  - name: home_volume_size
+    label: Home filesystem size (GB)
+    description: The size of the cloud volume or share to use for home directories
+    kind: integer
```

Comment:
Ah, I think this needs fixing up. But actually let's just remove this extra ui-meta now the azimuth change is almost merged to auto inject the correct configuration?